aidial_adapter_bedrock/chat_completion.py

```diff
@@ -21,22 +22,25 @@ def __init__(self, region: str):
     @dial_exception_decorator
     async def chat_completion(self, request: Request, response: Response):
         params = ModelParameters.create(request)
-        model_id = BedrockDeployment.from_deployment_id(
+        deployment_id = BedrockDeployment.from_deployment_id(
             request.deployment_id
-        ).model_id
-
-        model = await get_bedrock_adapter(
-            region=self.region,
-            model=model_id,
-            headers=request.headers,
         )
+        if deployment_id == BedrockDeployment.ANTHROPIC_CLAUDE_V3:
```

Comment:
I would rather put it under `get_bedrock_adapter` method.

In the `get_bedrock_adapter` you could match against `BedrockDeployment` instead of the model string to cover all of the cases explicitly.

Reply:
Done.